libsolidity/analysis/TypeChecker.cpp

```diff
@@ -408,17 +408,17 @@ void TypeChecker::endVisit(UsingForDirective const& _usingFor)
 		_usingFor.libraryName().annotation().referencedDeclaration
 	);
 	if (!library || !library->isLibrary())
-		m_errorReporter.fatalTypeError(_usingFor.libraryName().location(), "Library name expected.");
```

Comment:
Please either leave this fatal or try to crash the compiler in such a situation :-)